tests/rbd/rbd_utils.py

```diff
@@ -511,6 +520,29 @@ def image_unmap(self, device_name):
         """UnMap provided rbd image."""
         return self.exec_cmd(cmd=f"rbd unmap {device_name}", output=True)
 
+    def benchwrite(self, **kw):
+        """
+        This method writes IOs using rbd bench
+        Args:
+            **kw: test data
+                io: Amount of IO to be written
+                imagespec: pool/image on which IO needs to be run
+        """
+        if self.ceph_version < 3:
+            return self.exec_cmd(
+                cmd="rbd bench-write --io-total {} {}".format(
+                    kw.get("io", "500M"), kw.get("imagespec")
+                ),
+                long_running=True,
+            )
+        else:
+            return self.exec_cmd(
+                cmd="rbd bench --io-type write --io-threads 16 --io-total {} {}".format(
+                    kw.get("io", "500M"), kw.get("imagespec")
+                ),
+                long_running=True,
+            )
+
```

Comment:
`rbd bench-write`  commands seems to be deprecated
> # rbd help bench-write
> .....
> .......
> Simple write benchmark. (Deprecated, please use `rbd bench --io-type write` instead.)

Any particular reason for choosing it ? if not shall we avoid ?

Reply:
@VasishtaShastry I am not sure, I took this method directly from https://github.com/red-hat-storage/cephci/blob/master/tests/rbd_mirror/rbd_mirror_utils.py#L575-L589
Is it something like, for ceph version before 3, we need to use bench-write and for versions above 3 we use just bench?

Any idea?

Reply:
Made the changes as discussed, I have added an option to use the bench method for both read and write operations based on input arguments specified.
@VasishtaShastry could you please review and let me know if this looks okay?